pkgs/development/python-modules/nominal/default.nix

```diff
@@ -0,0 +1,97 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  hatchling,
+  requests,
+  nominal-api,
+  nominal-api-protos,
+  python-dateutil,
+  conjure-python-client,
+  pandas,
+  typing-extensions,
+  click,
+  pyyaml,
+  tabulate,
+  ffmpeg-python,
+  ruff,
+  mypy,
+  pytest,
+  pytest-cov,
+  pytestCheckHook,
+  polars,
+  pandas-stubs,
+  types-protobuf,
+  types-python-dateutil,
+  types-pyyaml,
+  types-requests,
+  types-tabulate,
+}:
+buildPythonPackage rec {
+  pname = "nominal";
+  version = "1.58.0";
+  pyproject = true;
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "nominal-io";
+    repo = "nominal-client";
+    tag = "v${version}";
+    hash = "sha256-GPwxDUHZSqlAQQrDiPO++/lRtW271mnawyiAnjoNMoM=";
+  };
+  build-system = [ hatchling ];
+  nativeCheckInputs = [
+    polars
+    ruff
+    pytest
+    pytest-cov
```

Comment:
```suggestion
    pytest-cov-stub
```

Reply:
If I use `pytest-cov-stub` then I also need to include `coverage` -- not a problem but wanted to flag for your PR with the note on testing.

Reply:
with just `pytest-cov-stub`, I got:
```
       > Executing pythonRuntimeDepsCheck
       > Checking runtime dependencies for nominal-1.58.0-py3-none-any.whl
       >   - coverage not installed
       For full logs, run:
```

Reply:
So `coverage` needs to be in `dependencies` anyway: https://github.com/nominal-io/nominal-client/blob/v1.58.0/pyproject.toml#L38, also `coverage.optional-dependencies.toml`.
But I feel like that's an upstream error. You should file an issue/open a PR.

Reply:
moved `coverage` to a dev dependency upstream, so now this is correctly in `nativeCheckInputs`